superface/super.json

```diff
@@ -297,12 +297,21 @@
         },
         "hubspot": {
           "file": "../capabilities/crm/contacts/maps/hubspot.suma"
+        },
+        "mixpanel": {
+          "file": "../capabilities/crm/contacts/maps/mixpanel.suma"
         }
-      }
+      },
+      "priority": [
```

Comment:
Maybe remove priority 🤷

Reply:
Shouldn’t be priority added by CLI only of there is more than one provoder configured? 

What it changes if it is set?

How sdk behaves when concrete provoder is passed in runtime? Does it failover anyway?

Reply:
I will look into CLI behavior.

It changes nothing, it's just dead code. You can keep it, my OCD kicked in 😅

Reply:
Ok, I am curious as it is what users will have in super.json as I am not expecting, they will be removing it.

Reply:
Hmm when I went to fix it, I noticed there is more providers. When I specify priority and omit some providers does it meant fallback to them won't happen?

For sure this is under documented.